docs/Reply.md

```diff
@@ -109,6 +110,16 @@ fastify.get('/', async function (req, rep) {
 Sets a response header. If the value is omitted or undefined, it is coerced
 to `''`.
 
+<a name="set-cookie"></a>
+- ### set-cookie
+    - While sending different values as cookie with `set-cookie` as the key, every value will be sent as cookie instead of replacing the previous value.
+
+    ```js
+    res.header('set-cookie', 'foo');
+    res.header('set-cookie', 'bar');
+    ```
+  - The browser will only consider the latest reference of a key for `set-cookie` header. The fact that this is done this way is to avoid parsing the set-cookie header when you add it in the reply and speeds up the serialization of the reply.
```

Comment:
Please mention if you need to reset the `set-cookie`, you need an explicit call to `reply.removeHeader('set-cookie')`.

Reply:
So I added a point for this that you need to make an explicit call to `reply.removeHeader('set-cookie')` and added a hyperlink to `.removeHeader` section there.